lang/pt.js

```diff
@@ -31,7 +31,7 @@ export default {
 	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {O número deve ser maior que {min}.} other {O número deve ser maior que ou igual a {min}.}}",
 	"components.form-element.input.text.tooShort": "{label} precisa ter, pelo menos, {minlength} caracteres",
 	"components.form-element.input.url.typeMismatch": "URL inválido",
-	"components.form-element.valueMissing": "{label} é obrigatório.",
+	"components.form-element.valueMissing": "O campo {label} é obrigatório.",
```

Comment:
"{label} is required" -> "The {label} field is required" 🤷

Reply:
This change might actually be more resilient to gender variances. `{label}` could be masculine or feminine depending on the term, but "field" would be more constant? 🤷

Reply:
I'm not sure. Seems (based on Google Translate) like there could be issues either way. 🤷 

"The date field is required" -> "O campo data é obrigatório"
"The age field is required" -> "O campo idade é obrigatório"
"The name field is required" -> "O campo nome é obrigatório"
"The score field is required" -> "O campo de pontuação é obrigatório"
"The content field is required" -> "O campo de conteúdo é obrigatório"

vs 

"Date is required" -> "A data é obrigatória"
"Age is required" -> "A idade é obrigatória"
"Name is required" -> "O nome é obrigatório"
"Score is required" -> "A pontuação é obrigatória"
"Content is required" -> "O conteúdo é obrigatório"

Reply:
But maybe it was ok that we excluded the "A" or "O" in our original?

Reply:
Yeah seems like there's no good answer here and we should have always done something like `Field is required: {label}`. I'd be fine if you wanted to just revert this back.